README.md

```diff
@@ -1,34 +1,36 @@
-# <div align="left"><img src="https://rapids.ai/assets/images/rapids_logo.png" width="90px"/>&nbsp;pynvjitlink</div>
+# <div align="left"><img src="img/rapids_logo.png" width="90px"/>&nbsp;pynvjitlink</div>
 
 The [RAPIDS](https://rapids.ai) pynvjitlink library provides a Python binding for the
-[nvJitLink library](https://docs.nvidia.com/cuda/nvJitLink/index.html). It is presently a
-work-in-progress.
+[nvJitLink library](https://docs.nvidia.com/cuda/nvJitLink/index.html).
 
-**NOTE:** For the latest stable [README.md](https://github.com/rapidsai/pynvjitlink/blob/main/README.md) ensure you are on the `main` branch.
+## Installation with pip
 
-## Quick Start
-TODO
+```shell
+pip install -i https://pypi.anaconda.org/rapidsai-wheels-nightly/simple pynvjitlink-cu12
```

Comment:
This is going to change soon. @raydouglass is working to publish the wheels on our public pypi.nvidia.com.

Reply:
Wheels are public now.
```suggestion
pip install --extra-index-url https://pypi.nvidia.com pynvjitlink-cu12
```